packages/venia-ui/lib/components/CartPage/PriceAdjustments/ShippingMethods/shippingForm.js

```diff
@@ -40,6 +42,7 @@ const ShippingForm = props => {
     return (
         <Fragment>
             <h3 className={classes.formTitle}>Destination</h3>
+            <FormError errors={formErrors} />
```

Comment:
You don't have to do this, but consider just rendering the `FormError` element as the first child of the `Form` element, rather than wrapping these forms in fragments. Might help if the form wants to have the form error participate in its layout.